### PR TITLE
Create ci.yml

### PR DESCRIPTION
**Pull request by crystal941**

### CODE REVIEW

**Review by crystal941 — COMMENTED** [review]
Review required

**Review by teemo36980 — APPROVED** [review]
Changes reviewed

**Review by teemo36980 — APPROVED** [review]

**Review by crystal941 — COMMENTED** [review]
confirm with changes

**Review by teemo36980 — APPROVED** [review]
reviewed

File: README.md [hunk 1]
```diff
@@ -14,50 +14,31 @@ _Create workflows that enable you to use Continuous Integration (CI) for your pr
...
-  Choose 3-5 steps for your course.
-  The first step is always the hardest, so pick something easy!
-  Link to docs.github.com for further explanations.
-  Encourage users to open new tabs for steps!
+  <<< Author notes: Step 4 >>>
+  Start this step by acknowledging the previous step.
+  Define terms and link to docs.github.com.
 -->
 
-## Step 1: Add a test workflow
```
Comment (teemo36980) [inline README.md:24]: change to step 4

**Review by teemo36980 — APPROVED** [review]
change to step 4

File: README.md [hunk 2]
```diff
@@ -14,50 +14,31 @@ _Create workflows that enable you to use Continuous Integration (CI) for your pr
...
-   - name: Run markdown lint
-     run: |
-       npm install remark-cli remark-preset-lint-consistent
-       npx remark . --use remark-preset-lint-consistent --frail
-   ```
-   > Even after the code is indented properly in `ci.yml`, you will see a build error in GitHub Actions. We'll fix this in the next step.
-1. Click **Start commit**, and choose to make a new branch named `ci`.
-1. Click **Propose a new file**.
-1. Click **Create pull request**.
-1. Wait about 20 seconds then refresh this page (the one you're following instructions from). [GitHub Actions](https://docs.github.com/en/actions) will automatically update to the next step.
```
Comment (teemo36980) [inline README.md:60]: wait message

**Review by teemo36980 — APPROVED** [review]
approved

**Review by crystal941 — COMMENTED** [review]
agreed

**Review by teemo36980 — APPROVED** [review]
updated

**Review by crystal941 — COMMENTED** [review]
reviewed

**Review by teemo36980 — APPROVED** [review]
approved

**Review by crystal941 — COMMENTED** [review]
#1

**Review by teemo36980 — APPROVED** [review]
reviewed

**Pull request merged**